### DISCUSSION TITLE
Milvus 2.2.0 Standalone Installation Issue: Dependency Failure for Server

### DISCUSSION THREAD

**Discussion by nitingangyan** [milvus-io/milvus#29886] [Q&A and General discussion]
Hello Team,

I successfully installed Milvus 2.2.0 standalone on my system, and it was functioning properly. However, it has abruptly stopped working in the past few days, and despite reinstalling it, I am encountering the same issue. I would appreciate your assistance in resolving this issue.

Details:

Milvus Version: 2.2.0 standalone
Error: Dependency failed for Milvus standalone server

Thanks...

**Comment by yhmo** [comment]
Where did you get the error "Dependency failed for Milvus standalone server"

**Reply by yhmo** [reply]
v2.3.x is recommended, 2.2.0 is pretty old.

**Reply by nitingangyan** [reply]
Thanks I will try, could you please share **rpm** file link for v2.3.x?

**Reply by yhmo** [reply]
We don't provide rpm installation anymore for v2.3.
We only provide: Docker, Helm, Operator

**Reply by nitingangyan** [reply]
ok thanks I will try with these options
